sam-bedrock-agent/README.md

```diff
@@ -0,0 +1,88 @@
+# SAM-BEDROCK-AGENT
+
+This plugin allows you to import one or multiple Amazon bedrock agents or flows as action to be used in your SAM project.
+
+
+
+## Add the Plugin
+
+If you haven't already, add the plugin to your SAM instance:
+```sh
+solace-agent-mesh plugin add sam-bedrock-agent --pip -u git+https://github.com/SolaceLabs/solace-agent-mesh-core-plugins#subdirectory=sam-bedrock-agent
+```
+
+## Using the Bedrock Agent
+
+The Amazon Bedrock Agent allows you to define custom actions within SAM that correspond to request/reply interactions over the Solace Event Mesh.
```

Comment:
I think this sentence still needs some work. This lets you integrate with Bedrock agents in AWS